Cargo.toml

```diff
@@ -25,7 +25,9 @@ hex = "0.4"
 k8s-openapi = { version = "0.18.0", features = ["v1_26"] }
 # disabling default features and enablinbg rustls to ensure openssl is not linked anywhere;
 # similar as for reqwest
-kube = { version = "0.82.2", default-features = false, features = ["rustls-tls"] }
+kube = { version = "0.82.2", default-features = false, features = [
```

Comment:
Random formatting changes in here?

Reply:
not sure why - is my rustfmt config different maybe?